build.gradle

```diff
@@ -3,16 +3,16 @@ plugins {
     id 'checkstyle'
     id 'pmd'
     id 'jacoco'
-    id 'com.github.johnrengelman.shadow' version '6.1.0'
```

Comment:
removing shadow i think is okay since we do not have a multimodule project with dependencies between those modules